Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:10 AS builder
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+RUN npm install
+RUN . /app/.env.production && npm run build
+
+FROM nginx:alpine AS server
```

Comment:
We could use the unpivileged, non-root image: https://hub.docker.com/r/nginxinc/nginx-unprivileged